CHANGELOG.md

```diff
@@ -3,14 +3,28 @@ All notable changes to this project will be documented in this file.
 
 The changelog format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## 24-Feb-2020
+
+**Milestone**: Fushicho.4(RC3 0.9.3.1)
+ Versions  |   |
+---|---|---
+SDK Core| v0.17.1 | https://www.npmjs.com/package/symbol-sdk
+Catbuffer | v0.0.11 | https://www.npmjs.com/package/catbuffer-typescript
+Client Library | v0.8.5  | https://www.npmjs.com/package/symbol-openapi-typescript-node-client
+
+- Rebranded `nem2-sdk` to `symbol-sdk`. Please be noted the package name changes.
+- **[BREAKING CHANGE]** Changed `QueryParameters` and `TransactionFilter` to use deconstructed argument (JSON object) in the constructor.
+- Added `node/peers` endpoint.
+- Fixed security issues reported by github.
+
 ## 17-Feb-2020
```

Comment:
Missing version number.

Format: `## [version] - date`

Reply:
The versions is placed underneath with table breakdown